platformcomponents/desktop/date-picker.json

```diff
@@ -0,0 +1,31 @@
+{
+  "date-picker": {
+    "comment": "Applied to date picker.",
+    "figma": "https://www.figma.com/file/NaNrfXjygZtRgMfHAFHjsp/Components---Windows?node-id=51360%3A117328",
+    "#normal": {
+      "background": "@theme-button-secondary-normal",
+      "text": "@theme-text-primary-normal",
+      "border": "@theme-outline-button-normal"
+    },
+    "#hovered": {
+      "background": "@theme-button-secondary-hover",
+      "text": "@theme-text-primary-normal",
+      "border": "@theme-outline-button-normal"
+    },
+    "#pressed": {
+      "background": "@theme-control-inactive-pressed",
+      "text": "@theme-text-primary-normal",
+      "border": "@theme-outline-button-normal"
+    },
+    "#disabled": {
+      "background": "@theme-button-secondary-pressed",
+      "text": "@theme-text-primary-disabled",
+      "border": "@theme-outline-button-normal"
+    },
+    "#selected": {
+      "background": "@theme-background-accent-normal",
+      "text": "@theme-common-text-white",
+      "border": "none"
+    }
```

Comment:
looks good, but there are some missing things. Seems like each element (background, text, border, badge...) has its own behaviour. Why not doing:

```
{
	"datePicker": {
		"comment": "(same)",
		"figma": "(same)",
		"date":{
			"background":{
				#normal: "@theme-button-secondary-normal",
				"#hovered": "@theme-button-secondary-hover",
				"#pressed": "@theme-button-secondary-pressed",
				"#active": "@theme-background-accent-normal"

			},
			"text": {
				"default": "@theme-text-primary-normal",
				"invalid": "@theme-text-primary-disabled",
				"active": "@theme-common-text-white"
			},
			"border":{
				"today": "@theme-outline-button-normal"
			}
			"unreadBadge": {
				"default": "@theme-background-accent-normal",
				'selected': "@theme-common-text-white"
			}
		}
	}
}
```